util/types/etc.go

```diff
@@ -49,10 +49,16 @@ func IsTypeChar(tp byte) bool {
 	}
 }
 
-//IsTypeSpecifiable returns a boolean indicating
-// whether the tp is a specifiable type.
-func IsTypeSpecifiable(tp byte) bool {
-	return IsTypeBlob(tp) || IsTypeChar(tp) || tp == mysql.TypeBit
+//IsTypePrefixable returns a boolean indicating
```

Comment:
Please leave a blank after `//`. The same as line 58.
@idlesummerbreeze Thanks for your PR!